src/find.js

```diff
@@ -39,7 +40,9 @@ module.exports = async function(collection, params) {
   // https://www.npmjs.com/package/mongoist#cursor-operations
   const findMethod = collection.findAsCursor ? 'findAsCursor': 'find';
 
-  const results = await collection[findMethod]({ $and: [cursorQuery, params.query] }, params.fields)
```

Comment:
Plz, sir. 
could you answer why did you remove the `await`?
I'm getting an error in line 45:
"collatedQuery.sort is not a function"

it happens because `query ` is a Promise

Reply:
It wasn't removed. It was moved to line 45.
`query` should be a `FindCursor`: [see docs](https://mongodb.github.io/node-mongodb-native/4.3/classes/Collection.html#find).
It's `toArray` that turns the `FindCursor` into a Promise: [docs](https://mongodb.github.io/node-mongodb-native/4.3/classes/FindCursor.html#toArray).